doc/board/xenguest_arm64.rst

```diff
@@ -0,0 +1,87 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+Xen guest ARM64 board
+=====================
+
+This board specification
+------------------------
+
+This board is to be run as a virtual Xen guest with U-boot as its primary
+bootloader. Xen [1] is a hypervisor that allows to manage virtual machines:
+install, enable and disable them. Xen's cross-platform and broad functionality
```

Comment:
```
Xen [1] is a hypervisor that allows to manage virtual machines:
￼install, enable and disable them.
```
not sure we need this as it doesn't describe the hypervisor in full...

Reply:
I think doc/board/xenguest_arm64.rst should be added as a dedicated (the last) commit: here we mix pv block and the board otherwise